lib/node_modules/@stdlib/math/base/special/sincospi/README.md

```diff
@@ -0,0 +1,78 @@
+# sincospi
+
+> Simultaneously compute the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a number times [π][@stdlib/math/constants/pi].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sincospi = require( '@stdlib/math/base/special/sincospi' );
+```
+
+#### sincospi( \[out,] x )
+
+Simultaneously computes the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a `number` times [π][@stdlib/math/constants/pi] more accurately than `sincos(pi*x)`, especially for large `x`.
+
+```javascript
+var v = sincospi( 0.0 );
+// returns [ ~0.0, ~1.0 ]
+
+v = sincospi( 0.5 );
+// returns [ ~1.0, ~0.0 ]
+
+v = sincospi( 0.1 );
+// returns [ ~0.309, ~0.951 ]
+
+v = sincospi( NaN );
+// returns [ NaN, NaN ]
+```
+
+By default, the function returns a two-element `array` containing sin(πx) and cos(πx). To avoid extra memory allocation, the function supports providing an output (destination) object.
+
+```javascript
+var out = new Float64Array( 2 );
+
+var v = sincospi( out, 0.0 );
+// return <Float64Array>[ ~0.0, ~1.0 ]
+
+var bool = ( v === out );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+```javascript
+var linspace = require( '@stdlib/math/utils/linspace' );
+var TWO_PI = require( '@stdlib/math/constants/float64-two-pi' );
```

Comment:
The require statement of `TWO_PI` is unused now and should be removed.

Reply:
That should have been caught by ESLint when linting Markdown. @rreusser have you initialized the git hooks?

Reply:
Or maybe revert the change in the README and update the example instead. TWO_PI as a canonical input value representing 360 degrees might be better than `2.0`?

Reply:
`2.0` is fine, given that `x = 2 => 2*pi`.

Reply:
Hmm… hooks are initialized but it didn't catch the readme issue. I know it has caught issues in the readme before. I'll reinitialize hooks and try again.

Reply:
Ah...I forgot. I disable the ESLint rule for this because the code blocks in READMEs have an implicit global (i.e., undefined variable in each code block): the main package export. So, the linter would not have caught this unfortunately. :(

Reply:
`<!-- eslint: global var1 -->` in the markdown? That'd be convoluted and probably not worthwhile 😄

Reply:
Yes, that would work, but would also be required for every code block in a README. We have well-over 1000 READMEs in the project. Hence, considerable work would be involved to get all the READMEs to pass lint once this rule would be turned on.

Reply:
However...it may be possible to programmatically update all the READMEs in a one off.